src/tendency.jl

```diff
@@ -10,15 +10,14 @@ function precompute_aux!(dY, Y, aux, t)
     @. aux.q_liq = TD.liquid_specific_humidity(aux.params, ts)
     @. aux.q_ice = TD.ice_specific_humidity(aux.params, ts)
     @. aux.T     = TD.air_temperature(aux.params, ts)
-
+    aux.w        = Geometry.WVector.(ones(FT, face_space))
+    @. aux.w     = aux.w * aux.w_params.w1 * sin(pi * t / aux.w_params.t1)
 end
```

Comment:
Would it make more sense to initialise `w` to zero in `KiD_driver` and here just write

`@. aux.w     = aux.w_params.w1 * sin(pi * t / aux.w_params.t1)`

Reply:
I tried that at first, but it complained about some typing with the Field Vector. Let me try something else.